Tools/Scripts/webkitpy/layout_tests/servers/websocket_server.py

```diff
@@ -116,11 +116,11 @@ def _prepare_config(self):
 
         # FIXME https://bugs.webkit.org/show_bug.cgi?id=206546: Should be using the same version of Python run-webkit-tests is
         python_interp = sys.executable
-        if sys.version_info > (3, 0):
-            python_interp = 'python2'
+        if sys.version_info < (3, 0):
```

Comment:
Seems like this unrelated change got included by mistake

Reply:
Not an unrelated change, a deliberate breakage I'm sending through the UAT instance to test this. Will not be landing this bit.

Reply:
https://ews-build.webkit-uat.org/#/builders/81/builds/26 should trigger this email